src/sentry/identity/oauth2.py

```diff
@@ -296,6 +296,13 @@ def exchange_token(self, request: Request, pipeline, code):
                 "error": "Could not verify SSL certificate",
                 "error_description": f"Ensure that {url} has a valid SSL certificate",
             }
+        except ConnectionError:
+            logger.info("identity.oauth2.json-error", extra={"url": self.access_token_url})
```

Comment:
`"identity.oauth2.json-error"` should probably be something like ``"identity.oauth2.connection-error"` on the logger now.